README.md

```diff
@@ -32,7 +32,7 @@ SaltedPassword = fast_scram:hi(Hash, Password, Salt, IterationCount)
 ```
 where `Hash` is the underlying hash function chosen as described by
 ```erlang
--type sha_type() :: crypto:sha1() | crypto:sha2().
+-type sha_type() :: crypto:sha1() | crypto:sha2() | crypto:sha3().
```

Comment:
As far as i understand, there's no `crypto:sha3()` type defined

Reply:
But there probably will very soon: https://github.com/erlang/otp/pull/9448. An this is a readme, so dialyzer won't fail to check it, that's why I thought it was ok to just use it here already.

Reply:
ok, I'm fine with it.